demos/demo-tf-search-compiled-graph.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, idx):
         :param int idx: hyp idx (to identify it in a beam)
         """
         self.idx = idx
-        self.source_idx = None  # type: typing.Optional[int]  # source hyp idx
+        self.source_idx: typing.Optional[int]  # source hyp idx = None
```

Comment:
```suggestion
        self.source_idx: typing.Optional[int]  = None  # source hyp idx
```
that looks like a bug in ruff formatter

Reply:
Nah, it's a bug in my search+replace :D Need to fix them.